lib/escape.js

```diff
@@ -53,6 +53,20 @@ exports.escapeHtml = function (input) {
 };
 
 
+exports.escapeJson = function (input) {
+
+    if (!input) {
+        return '';
+    }
+
+    return input.replace(/</g, '\\u003c')
```

Comment:
Yeah, that's looking about 3x faster: https://gist.github.com/g-k/c1efc1527cd74305b1e3cd272a49453a

Updated in: cf70d6d1dd8a6a71998b1d79ae64be069d301c7b